src/components/Icon/icons.js

```diff
@@ -1,229 +1,41 @@
-  file: '\\f13f',
-  fileFilled: '\\f140',
-  flashOn: '\\f16b',
-  forward: '\\f116',
-  genderNeutralUser: '\\f174',
-  googleLogoFilled: '\\f177',
-  grid: '\\f1ad',
-  guest: '\\f14d',
-  hashtag: '\\f182',
-  hashtagFilled: '\\f183',
-  help: '\\f105',
-  help2: '\\f106',
-  // DEPRECATED: use help2
-  help_2: '\\f106',
-  hide: '\\f1a7',
-  home: '\\f18a',
-  insertTable: '\\f13d',
-  journey: '\\f188',
-  journeyFilled: '\\f189',
-  link: '\\f193',
-  lock: '\\f147',
-  loginRounded: '\\f1a6',
-  logoutRounded: '\\f1a5',
-  lol: '\\f165',
-  lowPrice: '\\f170',
-  maintenance: '\\f18d',
-  menu: '\\f13a',
-  menu2: '\\f18f',
-  message: '\\f146',
-  minus: '\\f1bd',
-  moderator: '\\f14e',
-  more: '\\f13b',
-  moveGrabber: '\\f1a2',
-  msExcel: '\\f11a',
-  musicVideo: '\\f169',
-  ok: '\\f10a',
-  okFilled: '\\f10b',
-  onlineSupport: '\\f1a3',
-  openInWindow: '\\f191',
-  pdf2: '\\f15e',
-  // DEPRECATED: use pdf2
-  pdf_2: '\\f15e',
-  phone: '\\f141',
-  phoneFilled: '\\f142',
-  picture: '\\f166',
-  plusMath: '\\f181',
-  pokemon: '\\f163',
-  portrait: '\\f16a',
-  purchaseOrder: '\\f1bb',
-  readMessage: '\\f199',
-  restart: '\\f1ba',
-  save: '\\f1a1',
-  search: '\\f102',
-  searchFilled: '\\f107',
-  securityChecked: '\\f176',
-  server: '\\f195',
-  settings: '\\f1a9',
-  shoppingCart: '\\f1bf',
-  signature: '\\f1ae',
-  skypeFilled: '\\f178',
-  sms: '\\f184',
-  smsFilled: '\\f185',
-  sortingOptions: '\\f160',
-  speechBubble: '\\f143',
-  speechBubbleFilled: '\\f144',
-  speedometer: '\\f175',
-  spinnerFrame1: '\\f151',
-  spinnerFrame_1: '\\f151',
-  spinnerFrame2: '\\f152',
-  spinnerFrame_2: '\\f152',
-  spinnerFrame3: '\\f153',
-  spinnerFrame_3: '\\f153',
-  spinnerFrame4: '\\f154',
-  spinnerFrame_4: '\\f154',
-  spinnerFrame5: '\\f155',
-  spinnerFrame_5: '\\f155',
-  spinnerFrame6: '\\f156',
-  spinnerFrame_6: '\\f156',
-  spinnerFrame7: '\\f157',
-  spinnerFrame_7: '\\f157',
-  spinnerFrame8: '\\f158',
-  spinnerFrame_8: '\\f158',
-  stack: '\\f197',
-  stackOfPhotos: '\\f167',
-  thumbUp: '\\f16e',
-  toggleOff: '\\f10e',
-  toggleOnFilled: '\\f10f',
-  toolbox: '\\f18e',
-  trash: '\\f104',
-  tripadvisor: '\\f17b',
-  tvShow: '\\f16c',
-  uncheckedCheckbox: '\\f10c',
-  unicorn: '\\f14f',
-  up: '\\f19e',
-  up2: '\\f17c',
-  upload: '\\f19d',
-  user: '\\f15d',
-  userFemaleBody: '\\f15c',
-  userMaleCircle: '\\f100',
-  videoPlaylist: '\\f168',
-  viewMore: '\\f1bc',
-  visible: '\\f1a8',
-  voipGateway: '\\f198',
-  whatsappFilled: '\\f17a',
-  worker: '\\f14b',
-  xls: '\\f15f',
+  arrowDown: '\\f100',
+  arrowUp: '\\f101',
+  bell: '\\f102',
+  calendar: '\\f103',
+  caret: '\\f104',
+  cartEmpty: '\\f105',
+  cart: '\\f106',
+  checkmark: '\\f107',
+  code: '\\f108',
+  copy: '\\f109',
+  csv: '\\f10a',
+  delete: '\\f10b',
+  download: '\\f10c',
+  edit: '\\f10d',
+  envelope: '\\f10e',
+  expired: '\\f10f',
+  fileFilled: '\\f110',
+  file: '\\f111',
+  filterClear: '\\f112',
```

Comment:
Looks like clearFilters was renamed to filterClear, won't this break backwards compatibility?

Reply:
Yup. @braxtonCoats try to avoid changing the name of icons that bear the same general meaning as the previous set.

Reply:
Yes, good catch. I'll adjust that. Also many icons are being removed here to clean up things. I'm not 100% sure if any of the icons removed were used anywhere else.